core/standard/requirements/local-resources-catalogue/REQ_param-q-response.adoc

```diff
@@ -0,0 +1,6 @@
+[[req_local-resources-catalogue_param-q-response]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/local-resources-catalogue/param-q-response*
+^|A |If the `q` parameter is provided by the client, only local resources whose text fields contains one or more of the specified search terms SHALL be in the result set.
```

Comment:
The definition of the parameter is:
````
---
name: q
in: query
required: false
schema:
  type: array
  items:
    type: string
explode: false
style: form
````
so the search terms are comma-separated and so the **comma** delimits the terms.

Reply:
@pvretano Good point, on the other hand that's not what users expect, I think. Users type free text phrases with spaces into Google, I've never seen anyone type commas. This would eventually mean that you are not able to simply pass through the text field value, but instead you have to do something like `searchTerm.replace(' ', ',')` beforehand. That just doesn't feel right to me, but maybe it's just me?

Reply:
Moved the discussion to https://github.com/opengeospatial/ogcapi-records/issues/295